go/chat/helper.go

```diff
@@ -593,6 +610,30 @@ func newNewConversationHelper(g *globals.Context, uid gregor1.UID, tlfName strin
 	}
 }
 
+func (n *newConversationHelper) findConversations(ctx context.Context, membersType chat1.ConversationMembersType, topicName string) ([]chat1.ConversationLocal, []chat1.RateLimit, error) {
+	onechatpertlf := true
+	return FindConversations(ctx, n.G(), n.DebugLabeler, n.ri, n.uid, n.tlfName, n.topicType, membersType, n.vis, topicName, &onechatpertlf)
+}
+
+func (n *newConversationHelper) findExisting(ctx context.Context, topicName string) (res []chat1.ConversationLocal, rl []chat1.RateLimit, err error) {
+	// if IMPTEAM, check if a KBFS conversation exists already
+	if n.membersType == chat1.ConversationMembersType_IMPTEAM {
```

Comment:
I have preferred using `switch/case` for members type stuff, it just seems to fit better to me for some reason.